src/pds/registrysweepers/utils/__init__.py

```diff
@@ -250,3 +278,11 @@ def coerce_list_type(db_value: Any) -> List[Any]:
             db_value,
         ]
     )
+
+
+def get_human_readable_elapsed_since(begin: datetime) -> str:
```

Comment:
`humanize` provides a nice library for this, but this works too

Reply:
Seemed like a weak justification for a new dependency 😅